docker/prod/nodejs/nginx_staging.conf

```diff
@@ -2,6 +2,10 @@ upstream django_app {
   server django:8000 fail_timeout=0;
 }
 
+upstream statsd_exporter {
```

Comment:
Why are we changing nginx config if the `statsd_exporter` is not needed on the same instance as EvalAI backend?

Reply:
@Ram81 actually it would be good if we can keep the statsd container along with node-exporter container (as django backend) on the same container as metrics are streamed to statsd for each API call from the backend